erpnext/patches/v9_0/create_training_program_for_training_event.py

```diff
@@ -0,0 +1,26 @@
+# Copyright (c) 2017, Frappe and Contributors
```

Comment:
Lets not patch old Training Events. Lets keep Training Program Optional.

We will have to make this backward compatible, else we will have to change the version number.

Reply:
Ok @rmehta